### PR TITLE
fix: Cell now has default height and width of 1

### PR DESCRIPTION
**Pull request by zero0Halo**
**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [ ] Added a top level class to all my components `'.anchor-[COMPONENT NAME]'`.
- [x] Used [conventional commits](https://www.conventionalcommits.org) for all work.
- [x] Tested my solution on Mobile & Tablet.
- [ ] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [x] Updated snapshots for all permutations (`npm test -- -u`).
- [ ] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [ ] Created TODOs for known edge cases.
- [x] Documented all of my changes (inline & doc site).
- [ ] Made sure that all accessibility errors are resolved.
- [ ] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.
- [ ] Added name to OWNERS file for all new components
- [ ] If adding a new component, add its export to the rollup config
- [x] package.json version is bumped (if necessary)

---------
**Outline your feature or bug-fix below**

My code does...

### CODE REVIEW

File: src/Grid/Cell/Cell.component.tsx [hunk 1]
```diff
@@ -70,13 +70,13 @@ export class Cell extends React.PureComponent<CellProps> {
...
-        // fire a single time and before render.
+        // fire a single time and before render. Width and height have a default of 1.
         const { sortedResponsiveCSS, generalSettings } = generateBreakpointCSS(
             {
                 left: props.left,
-                height: props.height,
+                height: props.height || 1,
                 top: props.top,
-                width: props.width,
+                width: props.width || 1,
```
Comment (brandonlilly) [inline src/Grid/Cell/Cell.component.tsx:79]: What are the implications of the width defaulting to 1? Won't that mean by default the cell takes up 1/12 of the width of the grid? Is that the behavior that we want?

Comment (zero0Halo) [reply src/Grid/Cell/Cell.component.tsx:79]: Put it this way, it _was_ the intended behavior. The original library worked this way, and when I updated it for TypeScript I accidentally removed the default settings. 

The `Grid` itself has a default number of columns of 12, I think it's reasonable that cell's have a default width of 1. This is also easily overwritten via the `columns` prop in `Grid`. Its styles take precedence. For example, setting `columns` to `auto auto auto` still sets all `Cell`'s to auto width despite the default.

**Review by sharkparty — APPROVED** [review]
LGTM
